### PR TITLE
fix: Added SORA network controller

### PR DESCRIPTION
**Pull request by pgolovkin**
Added SORA network controller

### CODE REVIEW

File: src/chains-config/soraControllers.ts [hunk 1]
```diff
@@ -0,0 +1,33 @@
+import { ControllerConfig } from '../types/chains-config';
+
+/**
+ * SORA configuration fro sidecar.
```
Comment (TarikGul) [inline src/chains-config/soraControllers.ts:4]:
```suggestion
 * SORA configuration for sidecar.
```

**Review by TarikGul — APPROVED** [review]
lgtm, I tested it against `wss://mof3.sora.org` and it all checked out. Just one minor grumble with a comment spelling error.

**Comment by TarikGul** [comment]
Also before merging, @pgolovkin if you could run `yarn lint --fix` to fix the CI check. 

Thanks again.

File: src/chains-config/soraControllers.ts (hunk 1 above)
Comment (pgolovkin) [reply src/chains-config/soraControllers.ts]: fixed, thanks!

**Review by insipx — APPROVED** [review]

**Pull request merged**